tools/send_saved_searchs_notifications.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+## Only run on the production app
+if [ "$ENVIRONMENT" != "production" ];then
+  echo "L'envoi des courriels des alertes sur les recherches sauvegardées ne se fait qu'en production"
```

Comment:
```suggestion
  echo "L'envoi des courriels de notification pour les recherches sauvegardées ne se fait qu'en production"
```
nit: seulement pour être consistents sur la terminologie